companion.py

```diff
@@ -593,7 +601,7 @@ def __init__(
     ):
         super().__init__(query_time=query_time, play_sound=play_sound, auto_update=auto_update)
         self.message: str = message  # User-friendly reason for failure.
-        self.exception: int = exception  # Exception that recipient should raise.
+        self.exception: BaseException = exception  # Exception that recipient should raise.
```

Comment:
Use Exception here rather than BaseException if possible. BaseException implies things like SystemExit

Reply:
Yeah, I'll change that.  Blame the 'hint' from mypy.